From 14afa8933d40d1058fea08e441ec36d0f6393318 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@nxp.com>
Date: Tue, 6 Feb 2018 18:38:36 +0800
Subject: [PATCH 3349/5242] MLK-17515-2: ARM64: dts: move the pdn gpio to
 sound card node

commit  ba4d5dbd4c106d10942ad78d2a50fdf2f7edcfe9 from
https://source.codeaurora.org/external/imx/linux-imx.git

move the pdn gpio to sound card node for ak4458

Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
Reviewed-by: Cosmin Samoila <cosmin.samoila@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
index 316cf5c..0b08a1b 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
@@ -105,6 +105,7 @@
 		model = "ak4458-audio";
 		audio-cpu = <&sai1>;
 		audio-codec = <&ak4458_1>, <&ak4458_2>;
+		ak4458,pdn-gpio = <&gpio3 18 GPIO_ACTIVE_HIGH>;
 	};
 
 	sound-ak5558 {
@@ -585,7 +586,6 @@
 	ak4458_1: ak4458@10 {
 		compatible = "asahi-kasei,ak4458";
 		reg = <0x10>;
-		ak4458,pdn-gpio = <&gpio3 18 GPIO_ACTIVE_HIGH>;
 	};
 
 	ak4458_2: ak4458@12 {
-- 
1.7.9.5

